docs/source/Plugin/P008.rst

```diff
@@ -32,6 +32,8 @@ The value is placed in the Tag variable (this name can be changed).
 
 .. image:: P008_DeviceConfiguration.png
 
+* **Enable backward compatibility mode** When enabled, uses the GPIO pins in 'reversed' mode, to be compatible with previous versions of this plugin (by default it will be enabled when updating an existing device/task), to ensure the same Tag results are given as on previous ESPEasy releases. When unchecked (default when newly added), the scan results will be the same as received with other Wiegand protocol RFID & Keypad readers.

```

Comment:
Maybe this section also needs to indicate which build is the "previous". (or when it is fixed :) )